github_flow.md

```diff
@@ -0,0 +1,54 @@
+# Github flow
+
+## We have agreed, that we will work using Github flow. Simple instructions can be found below.
+
+In short words, we are forking repository from Code for Poznan organisation to our repository, push the changes there and create pull requests from our branches (eg. ``w1stler:sample_branch``) to ``codeforpoznan:master``. 
```

Comment:
you're sometimes using one ` and sometimes double ` for oneliners - can we choose one and stick to it?